docs/xshareclasses.md

```diff
@@ -439,11 +409,10 @@ behavior, which can improve the performance of class loading from the shared cla
 
         -Xshareclasses:nonfatal
 
-:   Allows the VM to start even if class data sharing fails. Normal behavior for the VM is to refuse to start if class data sharing fails. If you select `nonfatal` and the shared classes cache fails to initialize, the VM attempts to connect to the cache in read-only mode. If this attempt fails, the VM starts without class data sharing. See also [`fatal`](#fatal).
+:   Allows the VM to start, in most cases, even if class data sharing fails. Normal behavior for the VM is to refuse to start if class data sharing fails. If you select `nonfatal` and the shared classes cache fails to initialize, the VM attempts to connect to the cache in read-only mode. If this attempt fails, the VM starts without class data sharing. See also [`fatal`](#fatal).
```

Comment:
I still think we should mention: The `nonfatal` suboption doesn't protect against cache corruption caused by bugs, when the corruption isn't detected at the time the shared cache is opened.

Reply:
There is a note underneath that you might have missed. See:

***Note:** Unless it is important that your application runs with class data sharing, it is good practice to set this parameter. See [Creating a shared classes cache](shrc.md#creating-a-shared-classes-cache). However, cache corruption as a result of a bug in the operating system, VM, or user code might not be detected when opening the cache. In this situation, the cache is used and the application might crash.*

Is this ok?

Reply:
Looks good.